stack/base/before-notebook.d/40_prepare-aiida.sh

```diff
@@ -61,17 +62,15 @@ if [[ ${NEED_SETUP_PROFILE} == true ]]; then
     verdi computer configure core.local "${computer_name}" \
         --non-interactive                                               \
         --safe-interval 0.0
-fi
-
 
-# Show the default profile
-verdi profile show || echo "The default profile is not set."
+else
 
-# Make sure that the daemon is not running, otherwise the migration will abort.
```

Comment:
I got rid of this, since by definition when this script is run at startup the daemon cannot be running. It does make this script a bit less robust against future changes elsewhere, but since we control the stack I think it's fine (and issues here would be caught by tests)

Reply:
I got rid of it, we will now print the profile at the top if it exist.